src/db/schemas/settlement.model.ts

```diff
@@ -0,0 +1,82 @@
+import {
+  pgEnum,
+  pgTable,
+  real,
+  timestamp,
+  varchar,
+} from "drizzle-orm/pg-core";
+import { createInsertSchema, createSelectSchema } from "drizzle-zod";
+import type { z } from "zod";
+
+import { splitTypeArr } from "@/common/enums";
+
+import groupModel from "./group.model";
+import userModel from "./user.model";
+
+export const splitTypeEnum = pgEnum("splitType", splitTypeArr);
+
+const settlementModel = pgTable("settlement", {
+  id: varchar({ length: 60 })
```

Comment:
Now that I think, we should also mark id field as unique in every model just to be on the safer side.

We shall create a new issue for other models, but for this one please update it yourseld.

Reply:
`id` is already have primary key constraint, which means unique + not null. I removed `notNull` constraint also. Let me know if i am wrong here.

Reply:
You are right, just checked it👍🏻